src/components/QuranReader/ReadingView/index.tsx

```diff
@@ -128,6 +142,33 @@ const ReadingView = ({
           }}
         />
       </div>
+      <div className={styles.buttonsContainer}>
+        <Button
+          size={ButtonSize.Small}
+          className={styles.prevButton}
+          onClick={() => {
+            virtuosoRef.current.scrollToIndex({
```

Comment:
let's move the code outside to a scrollToNextPage() and scrollToPreviousPage.

Should add logging